spek-ide-plugin/intellij-common/src/main/kotlin/org/spekframework/intellij/synonym.kt

```diff
@@ -52,10 +53,17 @@ class PsiDescription(val annotation: PsiAnnotation) {
 
 class PsiDescriptions(val annotation: PsiAnnotation) {
     val sources: Array<PsiDescription> by lazy {
-        val value = annotation.findAttributeValue("sources")!! as PsiArrayInitializerMemberValue
-        val sources = value.initializers.map { it as PsiAnnotation }
-            .map(::PsiDescription)
-        sources.toTypedArray()
+        val value = annotation.findAttributeValue("sources")!!
+
+        if (value is PsiArrayInitializerMemberValue) {
+            value.initializers.map { it as PsiAnnotation }
```

Comment:
This cast is safe? Otherwise we could use filterIsInstance before

Reply:
cast if safe.